src/components/MenuItem.js

```diff
@@ -88,13 +89,13 @@ const MenuItem = (props) => {
             {({hovered, pressed}) => (
                 <>
                     <View style={[styles.flexRow, styles.pointerEventsAuto, styles.flex1, props.disabled && styles.cursorDisabled]}>
-                        {(props.icon && props.iconType === CONST.ICON_TYPE_ICON) && (
-                            <View
-                                style={[
-                                    styles.popoverMenuIcon,
-                                    ...props.iconStyles,
-                                ]}
-                            >
+                        <View
+                            style={[
+                                styles.popoverMenuIcon,
+                                ...props.iconStyles,
+                            ]}
+                        >
+                            {(props.icon && props.iconType === CONST.ICON_TYPE_ICON) && (
```

Comment:
By the looks of the props definitions, `props.icon` may be undefined, it is optional. This will render an empty
```jsx
<View
    style={[
        styles.popoverMenuIcon,
        ...props.iconStyles,
    ]}
>
```
if `props.icon` is not defined. It should have been:

```suggestion
                    {props.icon && (
                        <View
                            style={[
                                styles.popoverMenuIcon,
                                ...props.iconStyles,
                            ]}
                        >
                            {props.iconType === CONST.ICON_TYPE_ICON && (
```